gobblin-runtime/src/main/java/org/apache/gobblin/runtime/dag_action_store/MysqlDagActionStore.java

```diff
@@ -58,6 +64,8 @@ public class MysqlDagActionStore implements DagActionStore {
       + "flow_execution_id varchar(" + ServiceConfigKeys.MAX_FLOW_EXECUTION_ID_LENGTH + ") NOT NULL, "
       + "dag_action varchar(100) NOT NULL, modified_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP  on update CURRENT_TIMESTAMP NOT NULL, "
       + "PRIMARY KEY (flow_group,flow_name,flow_execution_id, dag_action))";
+  // Deletes rows older than retention time period (in seconds) to prevent this table from growing unbounded.
+  private static final String RETENTION_STATEMENT = "DELETE FROM %s WHERE modified_time < DATE_SUB(CURRENT_TIMESTAMP, INTERVAL ? SECOND)";
```

Comment:
the other class had:
```
DATE_SUB(CURRENT_TIMESTAMP(3), INTERVAL %s * 1000 MICROSECOND)
```
dunno which is preferable... but can we unify?

Reply:
For the other interval I used milliseconds because everything else was millisecond level precision anyway, I don't think it's necessary in this case. Both can be generalized to seconds if that's what you're suggesting unifying, but I was trying to avoid confusion of changing the units we use in regards to everything else in `MysqlMultiActiveLeaseArbiter`. They are slightly different statements anyway do they really need to be unified?

Reply:
if seconds makes more sense in one than ms, leave that.  but why should one use `%s` and another the bind var `?`?

Reply:
Ah that's a good point. Both can be bound using `%s` since the retention period is static for one deployment of the service.